espnet/nets/pytorch_backend/e2e_asr.py

```diff
@@ -135,6 +135,8 @@ def __init__(self, idim, odim, args, asr_model=None, mt_model=None):
         assert 0.0 <= self.mtlalpha <= 1.0, "mtlalpha should be [0.0, 1.0]"
         self.etype = args.etype
         self.verbose = args.verbose
+        # NOTE: for self.build method
+        args.char_list = getattr(args, "char_list", None)
         self.char_list = args.char_list
```

Comment:
self.char_list = getattr(...) is enough?

Reply:
no, because some classes in this `__init__` also requires the `args` to have `char_list`